buddyboss/buddyboss-platform
# PROD-926 #1307 - Add a profile type dropdown in the Profile Search form filed option

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1307 .

### How to test the changes in this Pull Request:

1. Enabled profile type from BuddyBoss > Settings > Profiles > Profile Types
2. Create few profile types in BuddyBoss > Profiles > Profile Types
3. Add a new field with profile type in the BuddyBoss > Profiles > Profile fields
4. Go to BuddyBoss > Profiles > Profile Search
5. Add field and select profile type field
6. Go to the website and check the profile type field that appear in the profile search form

### Proof Screenshots or Video
https://vimeo.com/630802529/12707f9d4d


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
